qa/suites/rados/basic/tasks/rgw_snaps.yaml

```diff
@@ -8,8 +8,8 @@ overrides:
         osd_max_omap_entries_per_request: 10
 tasks:
 - rgw:
-    default_idle_timeout: 3600
-    client.0: null
+    client.0:
+      default_idle_timeout: 3600
```

Comment:
@cbodley @liewegas I don't mind removing them altogether. I'm not sure we'd see the same issue in those tests, I think the problem we were seeing here was that the field was at the wrong level in the hierarchy.